src/main/logger/logger_workload.cpp

```diff
@@ -323,20 +324,15 @@ bool PrepareLogFile() {
 
   // Pick metrics based on benchmark type
   double throughput = 0;
-  double latency = 0;
```

Comment:
no good reason. simply because I haven't started implementing latency measurement in tpcc and ycsb benchmark. will do it in the future.